pytumblr/request.py

```diff
@@ -65,7 +68,10 @@ def post(self, url, params={}, files=[]):
             if files:
                 return self.post_multipart(url, params, files)
             else:
-                resp = requests.post(url, data=urllib.parse.urlencode(params), headers=self.headers, auth=self.oauth)
+                data = urllib.parse.urlencode(params)
+                if not PY3:
+                    data = bytes(data)
```

Comment:
Would `str(data)` be clearer? Or is there another reason for `bytes()` that I'm not aware of?

Reply:
I think `bytes()` is an alias to `str` in python2, and it's basically a step for the transition to python3 unicode strings. Since this code is specifically for python2, `str` might be more clear, so I'll go with that